extensions/elytron-security-jdbc/runtime/src/main/java/io/quarkus/elytron/security/jdbc/BcryptPasswordKeyMapperConfig.java

```diff
@@ -0,0 +1,76 @@
+package io.quarkus.elytron.security.jdbc;
+
+import org.wildfly.security.auth.realm.jdbc.mapper.PasswordKeyMapper;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+/**
+ * Configuration information used to populate a "bcrypt"
+ * {@linkplain org.wildfly.security.auth.realm.jdbc.mapper.PasswordKeyMapper}
```

Comment:
Same here, remove the link to a WildFly class, I guess the users should not really expect that the strong link with WildFly classes will remain

Reply:
Never mind, they are used directly in the code...